pkg/provisioners/inmemory/channel/reconcile.go

```diff
@@ -261,14 +261,7 @@ func (r *reconciler) listAllChannels(ctx context.Context) ([]eventingv1alpha1.Ch
 	channels := make([]eventingv1alpha1.Channel, 0)
 
 	opts := &client.ListOptions{
-		// TODO this is here because the fake client needs it. Remove this when it's no longer
-		// needed.
-		Raw: &metav1.ListOptions{
-			TypeMeta: metav1.TypeMeta{
-				APIVersion: eventingv1alpha1.SchemeGroupVersion.String(),
-				Kind:       "Channel",
-			},
-		},
+		Raw: &metav1.ListOptions{},
```

Comment:
Can you add a comment like `This is used to set the Continue option`? It's unclear at first glance why this is necessary.

Reply:
Done.

Reply:
Perfect thanks!